week-8/Homework/mandatory/3-dog-photo-gallery/exercise.js

```diff
@@ -0,0 +1,29 @@
+
+let button = document.getElementById("button");
+let ul = document.getElementById("ul");
+let li = document.createElement("li");
+let img = document.createElement("img");
+ul.appendChild(li);
+li.append(img);
```

Comment:
Would this work every time the button is clicked or would it only add a new list item once?